apps/editing-toolkit/editing-toolkit-plugin/global-styles/src/store.js

```diff
@@ -52,7 +52,7 @@ const actions = {
  * @param {string} optionsPath REST path used to interact with the options API.
  */
 export default ( storeName, optionsPath ) => {
-	registerStore( storeName, {
+	const store = createReduxStore( storeName, {
```

Comment:
I'd just declare `store` globally for the module and export it, then call `register()` just like we did for other PRs already. 

No need for the custom registration function IMHO.